lib/jwt_sessions/store_adapters/redis_store_adapter.rb

```diff
@@ -14,32 +14,32 @@ def initialize(token_prefix: JWTSessions.token_prefix, redis_client: nil, **opti
           @storage = redis_client
         else
           begin
-            require "redis"
+            require "redis-client"
             @storage = configure_redis_client(**options)
           rescue LoadError => e
-            msg = "Could not load the 'redis' gem, please add it to your gemfile or " \
+            msg = "Could not load the 'redis-client' gem, please add it to your gemfile or " \
                   "configure a different adapter (e.g. JWTSessions.store_adapter = :memory)"
             raise e.class, msg, e.backtrace
           end
         end
       end
 
       def fetch_access(uid)
-        csrf = storage.get(access_key(uid))
+        csrf = storage.call("GET", access_key(uid))
         csrf.nil? ? {} : { csrf: csrf }
       end
 
       def persist_access(uid, csrf, expiration)
         key = access_key(uid)
-        storage.set(key, csrf)
-        storage.expireat(key, expiration)
+        storage.call("SET", key, csrf)
+        storage.call("EXPIREAT", key, expiration)
```

Comment:
```suggestion
        storage.call("SET", key, csrf, ex: expiration)
```
redis-client allows setting expiration with a single call

Reply:
Thanks - all unit tests pass.